docs/_bug_rules.md

```diff
@@ -43,8 +43,8 @@ Once submitted, the issue will be responded to, verified, accepted, and rewarded
 
 ### Submission Branches
 Participants in the program are free to submit bugs on two different branches in the colonyNetwork codebase:
-* the current testnet release (tagged on the `master` branch) and deployed to Görli
-* against the `develop` branch where we merge ongoing work
+* the current testnet release (tagged on the `develop` branch) and deployed to Görli
+* against the `master` branch where we merge ongoing work
```

Comment:
Oh no, we don't merge ongoing work in `master`, that's where we release to mainnet from.